src/entity-manager/EntityManager.ts

```diff
@@ -178,7 +180,9 @@ export class EntityManager extends BaseEntityManager {
         } else {
             // todo: throw exception if constructor in target is not set
             if (target instanceof Array) {
-                return this.getRepository<Entity[]>(target[0].constructor).remove(entity as Entity[], options);
+                return Promise.all(target.map((t, i) => {
```

Comment:
You already merged it, but normally I don't mind updating! Don't be afraid to ask for things on PRs to lessen your load, even if you think they're minor :-)

Anyway, thanks for merging.